packages/network/src/createFastTxExecutor.ts

```diff
@@ -0,0 +1,139 @@
+import { ArgumentsType } from "vitest";
+import { BigNumber, Contract, Overrides, Signer } from "ethers";
+import { JsonRpcProvider } from "@ethersproject/providers";
+
+/**
+ * Create a stateful util to execute transactions as fast as possible.
+ * Internal state includes the current nonce and the current gas price.
+ */
+export async function createFastTxExecutor(
```

Comment:
I assume we'd ideally only instantiate one of these per signer? Otherwise lots of nonce issues? Is that worth noting here?